buildSrc/src/main/java/org/elasticsearch/gradle/info/GenerateGlobalBuildInfoTask.java

```diff
@@ -158,7 +156,7 @@ public void generate() {
                 writer.write("  Compiler JDK Version  : " + compilerJavaVersionEnum + " (" + compilerJavaVersionDetails + ")\n");
                 writer.write("  Compiler java.home    : " + compilerJavaHome + "\n");
                 writer.write("  Runtime JDK Version   : " + runtimeJavaVersionEnum + " (" + runtimeJavaVersionDetails + ")");
-                writer.write(inFipsJvm ? " (in FIPS 140 mode)\n":"\n");
+                writer.write(inFipsJvm ? " (in FIPS 140 mode)\n" : "\n");
```

Comment:
Let's move this to the `PrintGlobalBuildInfoTask` since it's more appropriate.

Reply:
I think whether or not we're in a FIPS 140 mode is a property of the JVM and were printing the current runtime version here, so it feels natural to add this here. WDYT ?

Reply:
That is true, but we aren't _actually_ making this determination based on the JVM. If we were, I'd agree, but since we're just toggling based on a system property, it doesn't make sense to include here. In fact, this will result in potential errors, because this generate task is cached, and we aren't tracking that system property as an input. So we expose ourselves to a scenario where we run a build, change the system property, and still get the old output.

Reply:
Gotcha, thanks for elaborating